src/script/auth/assets/WavesPattern.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Wire
+ * Copyright (C) 2025 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+export const WavesPattern = () => (
+  <svg
+    css={{
+      bottom: 0,
+      position: 'absolute',
+      width: '100%',
+      height: 'auto',
+      left: 0,
+    }}
+    width="457"
+    height="311"
+    viewBox="0 0 457 311"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      d="M841.651 -527.209C964.429 -307.492 902.996 -233.095 657.354 -304.019C411.713 -374.943 350.28 -300.546 473.058 -80.829C595.835 138.888 534.402 213.285 288.761 142.361C43.1188 71.4374 -18.3134 145.834 104.464 365.551C227.241 585.269 165.809 659.665 -79.8332 588.741"
+      stroke="#54A6FF"
```

Comment:
just to be 100%, the wave pattern is those colored lines on the bottom left?
In that case it's fine to have hardcoded color, just wanna be sure it's a part of the app without dark mode available